internal/operands/metrics/resources.go

```diff
@@ -152,6 +153,20 @@ var alertRulesList = []promv1.Rule{
 			componentAlertLabelKey:    componentAlertLabelValue,
 		},
 	},
+	{
+		Alert: Rhel6AlertName,
+		Expr:  intstr.FromString("sum by (namespace, name) (kubevirt_vm_rhel6) > 0"),
+		Annotations: map[string]string{
+			"summary": "VM {{ $labels.namespace }}/{{ $labels.name }} is based on RHEL6 template, and this will not be supported in the next release",
+			//"runbook_url": runbookURLBasePath + Rhel6AlertName,
```

Comment:
How can the runbook / `runbook_url` be created?

Reply:
Thanks! The runbook will be created under [kubevirt/monitoring](https://github.com/kubevirt/monitoring) project

Reply:
Can it be created in advance? Or only after the metric is available? (Sorry, I don't know much about runbooks)

Reply:
I don't know much neither :) In general, I think the two things can go in parallel. Personally, I think that documentation that says there is something, but in fact there is not, is worse than providing something without saying it. But I am open to other points of view. If you prefer to firstly update the documentation, no problem at all. WDYT?

Reply:
@fossedihelm every alerts must have a runbook in advance. Changing this test just to get the alert in the code is not good practice. The test was set in place for a reason.
Please revert the change to the test and add a runbook and runbook_url annotation to the alert asap. Thank you.

Reply:
Thanks @sradco https://github.com/kubevirt/monitoring/pull/177 for the runbook.
ssp changes ready as soon the monitoring pr is merged. Thanks and sorry